src/coreclr/tools/Common/Compiler/DependencyAnalysis/Target_X64/TargetRegisterMap.cs

```diff
@@ -32,6 +32,7 @@ public TargetRegisterMap(TargetOS os)
 
                 case TargetOS.Linux:
                 case TargetOS.OSX:
+                // TODO: case TargetOS.iOSSimulator:
```

Comment:
We should change this method to:

```
if (os == TargetOS.Windows)
{
...
}
else
{
...
}
```
It is unlikely that any OS outside Windows is going to use Windows calling convention.

Reply:
Is it ok to exclude the check for the supported system on x64 architectures?

Reply:
Yes

Reply:
Thank you, done.